product_customer_code/__manifest__.py

```diff
@@ -8,53 +8,40 @@
 ############################################################################
 #    Coded by: el_rodo_1 (rodo@vauxoo.com)
 ############################################################################
```

Comment:
Please use [this header](https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3)